app/dashboard/templates/profiles/profile.html

```diff
@@ -7,8 +7,14 @@
     {% endif %}
     {% include 'shared/head.html' %}
     {% include 'shared/cards_pic.html' %}
-    <link rel="stylesheet" href={% static "v2/css/dashboard.css" %} />
-    <link rel="stylesheet" href={% static "v2/css/profile.css" %} />
+	<link rel="stylesheet" href={% static "v2/css/dashboard.css" %} />
+	{% if is_my_profile and theme_type == 'light' %}
```

Comment:
i dont think you need `is_my_profile` here.. if a profile theme is light/dark, it should be shown for everyone

Reply:
Right, but it should not affect another person's profile when am viewing it with my settings. That's why the `is_my_profile` flag is there